SayTheirNames/Source/Utils/Color/UIColor+Resource.swift

```diff
@@ -26,6 +26,12 @@ import UIKit
 
 extension UIColor {
     
+    private static func dynamic(light: UIColor, dark: UIColor) -> UIColor {
+        return .init { trait in
+            return trait.isDarkModeOn ? dark : light
+        }
+    }
+    
```

Comment:
I don't know that this is clearer than just using the initializer that you're wrapping. It is also kind of limited. What if you wanted to add colors for high contrast?

Reply:
I added this so we could at least start adding dynamic colors to the app so they work during dark mode. definitely a lot of room to improve here in a different PR